### DISCUSSION TITLE
Disney Channel Germany HD

### DISCUSSION THREAD

**Discussion by MavijaviGit** [iptv-org/community#1107] [Q&A]
Does anyone have Disney Channel Germany in HD? Not from TVHeadend or Xtream

**Comment by Westh977** [comment]
Can You Guys Add Those To IPTV M3u Playlist Like 

List
WETA-DT3 WETA Kids
WETA-DT2 WETA UK
WETA-DT1 WETA HD
Universal Kids HD
Nick Jr. Too
NickToons
Family Channel
![dg4p2b2-6628e9aa-88d0-4200-bed4-b656421a2317](https://github.com/user-attachments/assets/1cd96b79-89c8-4051-9b90-6400ca19f62d)
![dg3cent-27530e74-f9d6-480b-b267-37730763bb4c](https://github.com/user-attachments/assets/eb792a0e-511f-4cda-80a9-96755332f61a)
![R (2)](https://github.com/user-attachments/assets/283cf77d-d008-40f8-9104-5fac85b12a60)
![WETA UK Logo](https://github.com/user-attachments/assets/209f708d-12cd-4b17-92e8-31e41a60ca52)
![Family_logo](https://github.com/user-attachments/assets/99465c7d-705a-4407-96e8-7a9df3e6c8cf)
Will You Guys Add These To A IPTV M3U Playlist Please Also, Can You Add Those Logos On There Too

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19924
